### PR TITLE
LPS-101720 SF Not thrown

### PR DESCRIPTION
**Pull request by csierra**
/cc @martamedio @arthurchan35

thx @martamedio!

### CODE REVIEW

**Comment by brianchandotcom** [comment]
See https://github.com/csierra/liferay-portal/pull/591

There are still just way too many inconsistencies. See my changes.

Also, type:

**Comment by brianchandotcom** [comment]
git grep "liferay-portlet:resourceURL id"

The naming is inconsistent. We have camelCase, or /xyz/abc_def

You're introducing a 3rd case.

**Comment by brianchandotcom** [comment]
@zsigmondrab @csierra fyi... 

You guys need to review every line for consistency before sending it to me.

I merge in about 80-100 pulls a day. I prioritize whatever is easiest and most consistent to ensure flow of all the teams.

The pulls for OTP have been difficult to merge because it costs me much more effort to review, so I always prioritize to the end of the day (or I risk not merging in the other stuff). If I don't have time at the end of the day, it gets pushed off to the next day.

Basically, review every every every line. Is there an existing pattern where it came from. If it breaks an existing pattern, then the pull gets delayed because merging in it will cause more problems in the long run.

Please resend.
